vault/resource_identity_group_policies.go

```diff
@@ -128,11 +91,7 @@ func identityGroupPoliciesRead(d *schema.ResourceData, meta interface{}) error {
 
 	resp, err := readIdentityGroup(client, id)
 	if err != nil {
-		// We need to check if the secret_id has expired
-		if util.IsExpiredTokenErr(err) {
```

Comment:
Should we strip this out?

Reply:
It doesn't really make sense to be here. I think it was just a copy/paste from another resource by the original author. If the user's Vault Token has expired, the whole operation should fail anyway.